install

```diff
@@ -40,6 +41,7 @@ build_benchmark=false
 rocm_path=/opt/rocm
 build_relocatable=false
 build_address_sanitizer=false
+build_freorg_bkwdcomp=true
```

Comment:
Shouldn't this be =false if the freorg-bkwdcomp command-line option turns it on?  Either that or have it the other way around...on by default and the command-line option turns it off...

Reply:
Updated flags in latest